app/sagas/init.js

```diff
@@ -29,71 +22,20 @@ export const initLocalSettings = function* initLocalSettings() {
 
 const restore = function* restore() {
 	try {
-		let hasMigration;
-		if (isIOS) {
-			hasMigration = yield AsyncStorage.getItem('hasMigration');
-		}
-
-		let { token, server } = yield all({
-			token: RNUserDefaults.get(RocketChat.TOKEN_KEY),
-			server: RNUserDefaults.get('currentServer')
-		});
-
-		if (!hasMigration && isIOS) {
-			let servers = yield RNUserDefaults.objectForKey(SERVERS);
-			// if not have current
-			if (servers && servers.length !== 0 && (!token || !server)) {
-				server = servers[0][SERVER_URL];
-				token = servers[0][TOKEN];
-			}
-
-			// get native credentials
-			if (servers) {
-				try {
-					// parse servers
-					servers = yield Promise.all(servers.map(async(s) => {
-						await RNUserDefaults.set(`${ RocketChat.TOKEN_KEY }-${ s[SERVER_URL] }`, s[USER_ID]);
-						return ({ id: s[SERVER_URL], name: s[SERVER_NAME], iconURL: s[SERVER_ICON] });
-					}));
-					const serversDB = database.servers;
-					yield serversDB.action(async() => {
-						const serversCollection = serversDB.collections.get('servers');
-						const allServerRecords = await serversCollection.query().fetch();
-
-						// filter servers
-						let serversToCreate = servers.filter(i1 => !allServerRecords.find(i2 => i1.id === i2.id));
-
-						// Create
-						serversToCreate = serversToCreate.map(record => serversCollection.prepareCreate(protectedFunction((s) => {
-							s._raw = sanitizedRaw({ id: record.id }, serversCollection.schema);
-							Object.assign(s, record);
-						})));
-
-						const allRecords = serversToCreate;
-
-						try {
-							await serversDB.batch(...allRecords);
-						} catch (e) {
-							log(e);
-						}
-						return allRecords.length;
-					});
-				} catch (e) {
-					log(e);
-				}
-			}
-
-			try {
-				yield AsyncStorage.setItem('hasMigration', '1');
-			} catch (e) {
-				log(e);
-			}
+		let token; let server;
+		try {
+			({ token, server } = yield all({
+				token: MMKV.getStringAsync(RocketChat.TOKEN_KEY),
+				server: MMKV.getStringAsync('currentServer')
```

Comment:
I think it's time for us to refactor this `currentServer` key already.
`RocketChat.CURRENT_SERVER`
WDYT?